src/dist_kmeans_init.h

```diff
@@ -198,6 +198,122 @@ namespace dist_custom {
         }
     };
 
+    // specialize dist_custom for kemans_init<parallelPlusDense>
```

Comment:
pls add a description how parallel++ differs from ++ method (or add a full description like for ++).